install_devconsole/consoledeveloper.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set +x
+OC_LOGIN_USERNAME=kubeadmin
+oc login -u ${OC_LOGIN_USERNAME} -p ${OC_LOGIN_PASSWORD}
```

Comment:
Why login? Why not use `KUBECONFIG`?

Reply:
We can do this as well.. thanks Baiju :)